docs/utilities/family.mdx

```diff
@@ -75,6 +75,20 @@ There are two ways to remove params.
   - shouldRemove is a function that takes two arguments `createdAt` in milliseconds and `param`, and returns a boolean value.
   - setting `null` will remove the previously registered function.
 
+### Workaround: Atom-in-atom
```

Comment:
Can we call something like an alternative approach instead of a workaround?

```suggestion
### Alternative approach: [Atom-in-atom](../guides/atoms-in-atom.mdx)
```

Reply:
Wait, it's not an atoms-in-atom pattern. It's simply a big atom and deriving atoms. Close to https://github.com/pmndrs/jotai/blob/main/docs/recipes/large-objects.mdx

```suggestion
### Alternative approach: A big atom and slicing atoms
```
(Not sure if slicing is the right word here.)